lib/shared/screens/admin/shared/components/options-list/option.jsx

```diff
@@ -101,15 +102,18 @@ export default class Option extends Component {
       OptionsList,
       value,
       white,
+      type,
       elementOverride,
       displayOverride
     } = this.props;
 
+    const optionValue = (type && type.toLowerCase() === 'boolean')? boolean(value):value;
```

Comment:
the value that is brought forward is a string, thats why i needed to check if the input type was intended to be a boolean. it comes from the input type